common/BUILD

```diff
@@ -41,28 +35,17 @@ java_library(
     tags = ["maven_coordinates=grakn.core:common:{pom_version}"],
 )
 
-genrule(
-    name = "grakn-version",
-    srcs = [
-        "templates/util/GraknVersion.java",
-        "//:VERSION",
-    ],
-    cmd = "VERSION=`cat $(location //:VERSION)`;sed -e \"s/VERSION_PLACEHOLDER/$$VERSION/g\" $(location templates/util/GraknVersion.java) >> $@",
-    outs = ["util/GraknVersion.java"]
-)
 
 deploy_maven_jar(
     name = "deploy-maven-jar",
-    targets = [":common"],
+    target = ":common",
+    package = "common",
+    deps = [
+        "//common/exception",
+        "//common/config",
+        "//common/http",
+        "//common/util"
+    ],
     version_file = "//:VERSION",
     deployment_properties = "//:deployment.properties",
 )
```

Comment:
Let's move `//:VERSION` and `//:deployment.properties` into `deploy_maven_jar()` as default values. You may override it if needed, but in general you should always use the one provided by the repository`

Reply:
In the end, we won't even need `//:VERSION` file as input, once we move `version` as a property in `//:deployment.properties`